### ISSUE TITLE
11 Audit and update the website images to meet IDL image ratio requirement

### ISSUE THREAD

**Issue by JennySanchez** [carbon-design-system/carbon-for-ibm-dotcom-website#797]
#### Objectives

We need to do an audit of the image set up (aspect ratio) on the website. Mike mentioned that not all the images on the website are following the IDL ratio best practices. This is an IDL violation.

📍 All the image need to follow the ratios listed below (actual content can be viewed on the IDL website [here](https://www.ibm.com/design/language/photography/tips-and-techniques#aspect-ratio))

![Screen Shot 2020-12-20 at 3.12.51 PM.png](https://images.zenhubusercontent.com/5da0db0a9f0a9500014f53bf/adfb59fe-0185-441d-99cb-f2d7a8d11290)

📍 An example Mike provided for us to take a look at is the [Expressive Theme page](https://ibmdotcom-library.s3-web.us-south.cloud-object-storage.appdomain.cloud/deploy-previews/753/guidelines/expressive-theme/).

#### Task:

`Please feel free to reach out to Jenny before starting the task for any questions you may have.`

- [ ] Audit all the images across the Carbon for IBM.com website
- [ ] Create a list of the images we currently have on the website and mark the ones we need to make an update with(Airtable App is recommended for this task).

#### Acceptance criteria

- [ ] The audit list is ready to be reviewed with the DDS leadership team
- [ ] Audit task issues created for each website pages audit for image update ready to be prioritized

**Comment by JennySanchez** [comment]
@wonilsuhibm This task is based on the Slack conversation Mike started with you and I. We need to have a discussion before the task can be started.

**Comment by RobertaJHahn** [comment]
Can be closed.  Will be managed as a sub-task to future work.
